database/factories/AttributeFactory.php

```diff
@@ -26,8 +27,8 @@ public function definition()
         return [
             'collection_id' => Collection::factory(),
             'token_id' => Token::factory(),
-            'key' => fake()->unique()->word(),
-            'value' => fake()->text(),
+            'key' => HexConverter::stringToHexPrefixed(fake()->unique()->word()),
+            'value' => HexConverter::stringToHexPrefixed(fake()->text()),
```

Comment:
We do, there is a unique index of:
collection-token-key
If we dont store it as hex it will throw a duplicated entry...

Reply:
The `key` was the unique part of the index not the `value`, the `value` can be duplicated as long as there's a unique `key`.  But as discussed internally, since the chain data might not always be a string but could in face be any bytes it would be safer to store as hex as well.